pytket/tests/circuit_test.py

```diff
@@ -274,12 +285,12 @@ def test_circuit_gen_ids() -> None:
     c.Measure(a[3], b[3])
     c.Measure(a[1], b[1])
 
-    assert c.n_qubits == 4
-    assert c._n_vertices() == 23
-    assert c.n_gates == 7
+    assert c.n_qubits == 4  # noqa: PLR2004
```

Comment:
I wonder if we should disable this warning for the whole `tests` directory.

Reply:
Done